11-dynamic-inventory/README.md

```diff
@@ -8,41 +8,39 @@ This exercise will introduce you to using dynamic inventories instead of static
+Before jumping into how to use the Dynamic Inventory plugin with a playbook, it's good to get to know what configuration options the plugin has and how you can group nodes given the information available from the data source (Docker in this case).
 
-First let's create the configuration file for the inventory plugin:
+1. First let's add the configuration for the inventory plugin to `local.docker.yml`:
 
 ```yaml
 #local.docker.yml
 plugin: community.docker.docker_containers
 docker_host: unix://var/run/docker.sock
 ```
 
-We can play around with the `ansible-inventory` CLI to understand what kind of inventory it creates:
+2. We can now play around with the `ansible-inventory` CLI to understand what kind of inventory it creates. Run the commands below:
 
 ```bash
-ansible-inventory -i local-docker.yml --graph
+ansible-inventory -i local.docker.yml --graph
 # OR
-ansible-inventory -i local-docker.yml --list
+ansible-inventory -i local.docker.yml --list
 ```
 
-The graph output is more concise out of the two, you'll probably want to use it most of the time to just check groups and the associated hosts.
-
-You will notice that there are few groups, but none really work for us currently if we want to target all the `ansible_katas_host_x` containers with our playbook. In order to do that we need to modify the configuration file `local.docker.yml` to dynamically build a convenient group name.
+You will notice that there are few groups, but none really work for us currently if we want to target only the `ansible_katas_host_x` containers with our playbook. In order to do that we need to modify the configuration file `local.docker.yml` to dynamically build a convenient group name.
 
 ### Dynamic Groups
 
-One of the challenges with Docker itself is that it does not have a concept like tags which is common way to group and query hosts/nodes in cloud providers such as AWS or Azure. For the purpose of this exercise we use a special environment variable `GROUP` which is injected into the `ansible_katas_host_x` containers by `docker-compose` during bootup.
+One of the challenges with Docker itself is that it does not have a concept like tags which is common way to group and query hosts/nodes in cloud providers such as AWS or Azure. So for the purpose of this exercise we will use a special environment variable `GROUP` which is injected into the `ansible_katas_host_x` containers by `docker-compose` during bootup.
 
-Modify the `local.docker.yml` to lookup the environment variables for all the containers it finds and then create groups named based on the environment variables:
+3. Modify the `local.docker.yml` to lookup the environment variables for all the containers it finds and then create groups named based on the environment variables:
```

Comment:
> create groups named based on the environment variables:

Maybe this should be "create groups with names based on the environment variables:" or "naming" instead of "names". 🤔 The original looks like some kind of brain fart from me.

Reply:
yeah good catch